.storybook/preview.js

```diff
@@ -0,0 +1,6 @@
+// import { addParameters } from '@storybook/react'; // <- or your storybook framework
```

Comment:
Do you still need this file?

Reply:
I kept this file just in case any user wanted to try the global config, and to show proof of the example. It won't be compiled as part of the dist